doc/contributing/sw/adding_python_depedencies.md

```diff
@@ -4,5 +4,5 @@ To comply with software supply chain security requirements of various project pa
 We accomplish this with the help of the `uv pip compile` command, which is part of the `uv` package.
 
 If you need to add another Python package to the project, do so by:
-1. adding the package and version number to the `pyproject.toml` file, in the form of `<package>==<version>`, and
+1. adding the package and version number to the `pyproject.toml` file, in the form of `<package> ~= <version>` (or `<pacakge> == <version>` in case a specific version is needed), and
```

Comment:
```suggestion
1. adding the package and version number to the `pyproject.toml` file, in the form of `<package> ~= <version>` (or `<package> == <version>` in case a specific version is needed), and
```

Reply:
Fixed